client/jetpack-connect/remote-credentials.js

```diff
@@ -0,0 +1,90 @@
+/** @format */
+/**
+ * Component which handle remote credentials for installing Jetpack
+ */
+import React, { Component } from 'react';
+import { noop } from 'lodash';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * External dependencies
+ */
+import CredentialsForm from 'components/credentials-form';
+import FormattedHeader from 'components/formatted-header';
+import HelpButton from './help-button';
+import JetpackLogo from 'components/jetpack-logo';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import MainWrapper from './main-wrapper';
+
+export class OrgCredentialsForm extends Component {
+	getHeaderImage() {
+		return (
+			<div className="jetpack-connect__auth-form-header-image">
+				<JetpackLogo full size={ 45 } />
+			</div>
+		);
+	}
+
+	getHeaderText() {
+		const { translate } = this.props;
+
+		return translate( 'Add your website credentials' );
+	}
+
+	getSubHeaderText() {
+		const { translate } = this.props;
+
+		return (
+			<div className="jetpack-connect__install-step">
+				{ translate(
+					'Add your WordPress administrator credentials' +
+						'for this site. Your credentials will not be stored and are used for the purpose' +
+						'of installing Jetpack securely. You can also skip this step entirely and install Jetpack manually.'
+				) }
+			</div>
+		);
+	}
+
+	handleHelpButtonClick() {
+		//do something
+	}
+
+	footerLink() {
+		return (
+			<LoggedOutFormLinks>
+				<LoggedOutFormLinkItem>
+					{ this.props.translate( 'Install Jetpack manually.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Install Jetpack manually' )` *ES Score: 18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/jp/remote-install-creds-route)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).